pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -813,3 +813,34 @@ func MergeMaps(a, b map[string]string) map[string]string {
 	}
 	return a
 }
+
+// IsEnvUpdated looks for new env vars in the old env vars and returns true if
+// new env vars are not found
+func IsEnvUpdated(old, new []corev1.EnvVar) bool {
```

Comment:
its easier to simply build a map and compare them using reflect.DeepEqual()

Reply:
done, PTAL